core/domain/user_jobs_one_off_test.py

```diff
@@ -1896,12 +1900,12 @@ def test_standard_operation(self):
                 taskqueue_services.QUEUE_NAME_ONE_OFF_JOBS), 1)
         self.process_and_flush_pending_tasks()
 
-        self.assertEqual(
-            len(user_models.UserSubscriptionsModel.get(self.owner_id)
-                .activity_ids), 0)
-        self.assertEqual(
-            len(user_models.UserSubscriptionsModel.get(self.user_id)
-                .activity_ids), 0)
+        self.assertEqual(len(
+            user_models.UserSubscriptionsModel.get(self.owner_id)
```

Comment:
For cases like this (where it's a bit awkward because the clause is too long to fit in one line), just declare a new variable above it:

```
owner_subscriptions_model = user_models.UserSubscriptionsModel.get(self.owner_id)
self.assertEqual(len(owner_subscriptions_model.activity_ids), 0)
```

Ditto elsewhere.